packages/react-pdf-dom/src/Document.js

```diff
@@ -0,0 +1,58 @@
+/* global URL */
+import React, { Component, PropTypes } from 'react';
+import { PDFRenderer, createElement, pdf } from 'react-pdf';
+
+class Document extends Component {
+  container = createElement('DOCUMENT');
+
+  static propTypes = {
+    children: PropTypes.any,
+    height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    width: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.renderer = pdf();
+
+    this.state = {
+      document: undefined,
+    };
+  }
+
+  componentDidMount() {
+    this.mountNode = PDFRenderer.createContainer(this.container);
+    PDFRenderer.updateContainer(this.props.children, this.mountNode, this);
+    this.renderer.toBlob(this.container);
```

Comment:
Not really, because you don't have to talk to the public `PDFRenderer.render()` method for making it to work within react-dom and react-native. Took this from the react-art renderer, and how they did it.